src/components/AccessibilityPolicyPage/index.jsx

```diff
@@ -3,6 +3,8 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 // import styles from './AccessibilityPolicyPage.scss';
+import AccessibilityBody from '../AccessibilityBody';
```

Comment:
I just realized that the form is probably technically part of the "body", but I think that's probably super inconsequential.